src/components/graph/menus/network-modification-node-editor.tsx

```diff
@@ -249,72 +255,66 @@ const NetworkModificationNodeEditor = () => {
-        return withDefaultParams(Dialog, nprops);
-    }
-
-    const menuDefinition = [
+    const menuDefinition: MenuDefinition[] = [
         {
             id: 'CREATE',
             label: 'menu.create',
             subItems: [
                 {
                     id: MODIFICATION_TYPES.SUBSTATION_CREATION.type,
                     label: 'SUBSTATION',
-                    action: () => adapt(SubstationCreationDialog),
+                    action: () => withDefaultParams(SubstationCreationDialog),
                 },
                 {
                     id: MODIFICATION_TYPES.VOLTAGE_LEVEL_CREATION.type,
                     label: 'VOLTAGE_LEVEL',
-                    action: () => adapt(VoltageLevelCreationDialog),
+                    action: () => withDefaultParams(VoltageLevelCreationDialog),
                 },
                 {
                     id: MODIFICATION_TYPES.LINE_CREATION.type,
                     label: 'LINE',
-                    action: () => adapt(LineCreationDialog),
+                    action: () => withDefaultParams(LineCreationDialog),
                 },
                 {
                     id: MODIFICATION_TYPES.TWO_WINDINGS_TRANSFORMER_CREATION.type,
                     label: 'TWO_WINDINGS_TRANSFORMER',
-                    action: () => adapt(TwoWindingsTransformerCreationDialog),
+                    action: () => withDefaultParams(TwoWindingsTransformerCreationDialog),
                 },
                 {
                     id: 'GENERATOR_CREATION',
                     label: 'GENERATOR',
-                    action: () => adapt(GeneratorCreationDialog),
+                    action: () => withDefaultParams(GeneratorCreationDialog),
                 },
                 {
                     id: MODIFICATION_TYPES.LOAD_CREATION.type,
                     label: 'LOAD',
-                    action: () => adapt(LoadCreationDialog),
+                    action: () => withDefaultParams(LoadCreationDialog),
                 },
                 {
                     id: MODIFICATION_TYPES.BATTERY_CREATION.type,
                     label: 'BATTERY',
-                    action: () => adapt(BatteryCreationDialog),
+                    action: () => withDefaultParams(BatteryCreationDialog),
                 },
                 {
                     id: MODIFICATION_TYPES.SHUNT_COMPENSATOR_CREATION.type,
                     label: 'ShuntCompensator',
-                    action: () => adapt(ShuntCompensatorCreationDialog),
+                    action: () => withDefaultParams(ShuntCompensatorCreationDialog),
                 },
                 {
                     id: MODIFICATION_TYPES.VSC_CREATION.type,
                     label: 'VSC',
-                    action: () => adapt(VscCreationDialog),
+                    action: () => withDefaultParams(VscCreationDialog),
                 },
             ],
         },
         {
             id: 'CREATE_MULTIPLE',
```

Comment:
add enum for CREATE, CREATE_MULTIPLE, etc similar to copy type?

Reply:
same answer here